test/views/sections/items/ItemCommodityCodeViewSpec.scala

```diff
@@ -47,7 +47,7 @@ class ItemCommodityCodeViewSpec extends ViewSpecBase with ViewBehaviours {
       val view = app.injector.instanceOf[ItemCommodityCodeView]
       val form = app.injector.instanceOf[ItemCommodityCodeFormProvider].apply()
       val submitRoute = controllers.sections.items.routes.ItemCommodityCodeController.onPageLoad(testErn, testDraftId, testIndex1, NormalMode)
-      val testEpc = "testEpc"
+      val testEpc = "testEpcTobacco"
```

Comment:
Does this need to change?
```suggestion
      val testEpc = testEpcTobacco
```